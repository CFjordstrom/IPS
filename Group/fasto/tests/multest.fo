fun int main() =
  let x = read(int) in
  let y = read(int) in
  let a = write(x*y) in
  let b = write(x*3) in
  write(5*x)