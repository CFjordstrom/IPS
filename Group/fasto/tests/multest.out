30
15
25
